vividus-plugin-browserstack/build.gradle

```diff
@@ -4,10 +4,8 @@ dependencies {
     implementation project(':vividus-engine')
     implementation project(':vividus-util')
     implementation project(':vividus-reporter')
-    implementation project(':vividus-soft-assert')
     implementation project(':vividus-extension-selenium')
-    implementation project(':vividus-reporter')
-    implementation project(':vividus-plugin-rest-api')
```

Comment:
it should be rather `runtime`, since we have only one implementation of `JsonContext`